app/views/foods/_food_list.html.erb

```diff
@@ -0,0 +1,20 @@
+<table class='table'>
+  <thead>
+    <tr>
+      <th>Nutrient Name</th>
+      <th>Amount</th>
+      <th>Derivation Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% if nutrients %>
+      <% nutrients.each do |nutrient| %>
+        <tr id='fdcId-<%=fdicId%>-nutrient-<%=nutrient[:nutrientId]%>'>
+          <td><%=nutrient[:nutrientName]%></td>
+          <td><%=nutrient[:value]%> <%=nutrient[:unitName]%></td>
+          <td><%=nutrient[:derivationDescription]%></td>
+        </tr>
+      <% end %>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
Nice partial here! This is extra in terms of what the challenge asked for, but I like it!